scylla_cdc.go

```diff
@@ -91,3 +96,23 @@ func (c scyllaCDCToken) Less(token token) bool {
 func (c scyllaCDCToken) AsInt64() int64 {
 	return int64(c)
 }
+
+func scyllaIsCdcTable(session *Session, keyspaceName, tableName string) (bool, error) {
+	if !strings.HasSuffix(tableName, scyllaCDCLogTableNameSuffix) {
+		// Not a CDC table, use the default partitioner
+		return false, nil
+	}
+
+	// Get the table metadata to see if it has the cdc partitioner set
+	keyspaceMeta, err := session.KeyspaceMetadata(keyspaceName)
+	if err != nil {
+		return false, err
+	}
+
+	tableMeta, ok := keyspaceMeta.Tables[tableName]
+	if !ok {
+		return false, ErrNoMetadata
+	}
+
+	return tableMeta.Options.Partitioner == scyllaCDCPartitionerFullName, nil
```

Comment:
Nice. We totally forgot about this implementing the support in Java driver and checked base table for cdc property but this is even better and more generic!